rxv/ssdp.py

```diff
@@ -62,7 +62,10 @@ def discover(timeout=1.5):
 def rxv_details(location):
     """Looks under given UPNP url, and checks if Yamaha amplituner lives there
        returns RxvDetails if yes, None otherwise"""
-    xml = ET.XML(requests.get(location).content)
+    try:
+        xml = ET.XML(requests.get(location).content)
+    except:
```

Comment:
Ah good point. This is the lack of familiarity with python showing--I wasn't sure if lower-level exceptions like that would be caught here. Agreed that in this case it's probably fine, but I'll keep this in mind for next time!